README.md

```diff
@@ -339,6 +339,13 @@ For example, if you are using an Adafruit Metro M4 Express, you would use `pio r
    - Arduino IDE: `Tools ► Serial Monitor` from the menu or `ctrl+shift+m` on your keyboard.
 2. Your RC channel values will be there, & they will change as you move the sticks on your RC handset.
 
+### Viewing telemetry
```

Comment:
I _may_ need to provide a screenshot from my RadioMaster here showing what telemetry should appear. This will help give folks an idea of what to expect.

Reply:
Update: Meh. This seems a little unnecessary.

To be fair, I was overthinking this when I wrote it. I have thought it through and come to the conclusion that it's minutiae that will benefit next-to-nobody.

The approach I have taken to CRSF for Arduino thus far is to _not_ bog myself down with minutiae.
If someone requests this (via an Issue), I can add it in. Until then, I don't see a point in guessing whether-or-not someone will benefit from something like this... only to find that the thing benefits nobody (and, by extension, is a colossal waste of my time that could have been spent doing something that actually matters to my project).